src/4.3_to_5.0/migrate_43_to_50.ts

```diff
@@ -211,33 +204,23 @@ const accumulateStructure = ({
 export async function migrate_43_to_50(
   legacyUIFolder: ContentRecord,
   {
+    migratedUIFolder,
```

Comment:
This shouldn't be needed.
Like every migration function, it should just mutate an input `contentServer`.